spec/fixtures/integration/expected_report.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.16.0",
+  "version": "2.17.0",
```

Comment:
Is this intentionally hard-coded?  Why can we do the following?
{
  "version":  Salus::VERSION,
  ....
}

@joshua-ostrom